<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateIteration</class>
 <widget class="QScrollArea" name="CreateIteration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>668</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>400</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>1</width>
    <height>1</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>750</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create an iteration</string>
  </property>
  <property name="widgetResizable">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="scrollAreaWidgetContents">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>773</width>
     <height>664</height>
    </rect>
   </property>
   <property name="sizeGripEnabled" stdset="0">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="Iteration_Name">
      <property name="text">
       <string>Iteration Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>117</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="3" colspan="2">
     <widget class="QLineEdit" name="LEName">
      <property name="minimumSize">
       <size>
        <width>290</width>
        <height>21</height>
       </size>
      </property>
      <property name="maxLength">
       <number>64</number>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="Iter_Parent">
      <property name="text">
       <string>Previous iteration</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="PBIterParent">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>27</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="3" colspan="2">
     <widget class="QLineEdit" name="LEIterationParentName">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>21</height>
       </size>
      </property>
      <property name="maxLength">
       <number>64</number>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="Mesh_n">
      <property name="text">
       <string>Mesh n</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <spacer name="horizontalSpacer_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>117</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="3" colspan="3">
     <widget class="QLineEdit" name="LEMeshName_n">
      <property name="minimumSize">
       <size>
        <width>290</width>
        <height>21</height>
       </size>
      </property>
      <property name="maxLength">
       <number>64</number>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="Mesh_np1">
      <property name="text">
       <string>Mesh n+1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <spacer name="horizontalSpacer_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>117</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="3" colspan="3">
     <widget class="QLineEdit" name="LEMeshName_np1">
      <property name="minimumSize">
       <size>
        <width>290</width>
        <height>21</height>
       </size>
      </property>
      <property name="maxLength">
       <number>64</number>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>62</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0" colspan="6">
     <widget class="QGroupBox" name="GBField">
      <property name="title">
       <string>Field information</string>
      </property>
      <layout class="QGridLayout">
       <property name="margin">
        <number>9</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="0" column="2" colspan="5">
        <widget class="QLineEdit" name="LEFieldFile">
         <property name="minimumSize">
          <size>
           <width>282</width>
           <height>21</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="3" colspan="4">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>138</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="6">
        <widget class="QRadioButton" name="RBChosen">
         <property name="text">
          <string>Chosen time step</string>
         </property>
        </widget>
       </item>
       <item row="4" column="6">
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>255</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="5">
        <widget class="QSpinBox" name="SpinBox_Rank">
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>1010000</number>
         </property>
         <property name="value">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QLabel" name="Rank">
         <property name="text">
          <string>Rank</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2" colspan="2">
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="SpinBox_TimeStep">
         <property name="minimum">
          <number>-2</number>
         </property>
         <property name="maximum">
          <number>100000</number>
         </property>
         <property name="value">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="TimeStep">
         <property name="text">
          <string>Time step</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="4">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>138</width>
           <height>28</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="3" colspan="3">
        <widget class="QRadioButton" name="RBLast">
         <property name="text">
          <string>Last time step</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QRadioButton" name="RBNo">
         <property name="text">
          <string>No time step</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="PushFieldFile">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="FieldFile">
         <property name="text">
          <string>Field file</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="6">
     <spacer name="horizontalSpacer_10">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>69</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="0">
     <spacer>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>22</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="0" colspan="4">
     <widget class="QGroupBox" name="Hypothese">
      <property name="title">
       <string>Hypothesis</string>
      </property>
      <layout class="QGridLayout">
       <property name="margin">
        <number>9</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QComboBox" name="CBHypothese">
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="PBHypoEdit">
         <property name="text">
          <string>Edit</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="PBHypoNew">
         <property name="text">
          <string>New</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="7" column="4" colspan="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="1">
     <spacer>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>21</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="9" column="0" colspan="4">
     <widget class="QGroupBox" name="GroupButtons">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout">
       <property name="margin">
        <number>9</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="buttonOk">
         <property name="text">
          <string>OK</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="buttonApply">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="buttonCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="buttonHelp">
         <property name="text">
          <string>Help</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="9" column="4" colspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="5">
     <spacer name="horizontalSpacer_9">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>139</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
